pyprocessing/math.py

```diff
@@ -1,10 +1,20 @@
 import math
 import random
-
-from pyprocessing.calculation import lerp
-
+import re
+from collections.abc import Iterable
```

Comment:
should be at the top of the file (alphabetical order for imports), and with 2 line breaks before the code